content/workshops/intro-to-css/_index.md

```diff
@@ -1,3 +1,44 @@
 ---
 title: Introduction to CSS
+ready: true
 ---
+
+## Design basics
+
+> There are two great days in a person's life
+>
+> - the day they are born and the day they discover why.
+
+The follow are to be demonstrated and addressed during the workshop
+
+### Principles of design
+
+1.  Contrasts
+2.  Balance
+3.  Typography
+4.  Hierarchy
```

Comment:
It seems like a lot of this content would be better here: topics/web-frontend/intro-to-web-design/

It looks like the only section that should be here is the CSS foundation section. The rest is design